vue/components/ui/molecules/button-icon-dropdown/button-icon-dropdown.vue

```diff
@@ -63,9 +63,16 @@ export const ButtonIconDropdown = {
         };
     },
     methods: {
+        open() {
+            this.dropdownVisible = true;
+        },
+        close() {
+            this.dropdownVisible = false;
+        },
         onButtonIconClick() {
             if (this.disabled) return;
             this.dropdownVisible = !this.dropdownVisible;
+            this.$emit(`click:${this.dropdownVisible ? "open" : "close"}`, this);

```

Comment:
Another way to do this would be to watch `dropdownVisible` and emit the value when it changes. [Example](https://github.com/ripe-tech/ripe-components-vue/blob/master/vue/components/ui/molecules/notification/notification.vue#L163)